main.py

```diff
@@ -7,34 +7,60 @@
 
 class Shot(Image):
     velocity = Vector(0, 0)
+    shot_speed = 600
+    def clone(self):
+        clone = Shot()
+        clone.id = self.id
+        clone.source = self.source
+        clone.allow_stretch = self.allow_stretch
+        clone.size = self.size
+        clone.center = self.center
+        return clone
+    
     def update(self, dt):
-        position = Vector(self.center) + self.velocity
+        position = Vector(self.center) + self.velocity * dt
         if (position.x < 0 or position.x > self.parent.width or position.y < 0 or position.y > self.parent.height):
             self.velocity = Vector(0, 0)
             position = Vector(self.parent.width / 2, 0)
         self.center = position
+        
+    def shoot(self, direction):
+        self.velocity = direction * self.shot_speed
 
 class SpawningScreen(Widget):
     shot = ObjectProperty(None)
+    shots = []
+    total_shots = 3
     angle = NumericProperty(0)
     target = Vector(0, 0)
+    def start(self, time):
+        self.shot.center = Vector(self.width / 2, 0)
+        for index in range(0, self.total_shots):
+            shot = self.shot.clone()
+            self.shots.append(shot)
+            self.add_widget(shot, len(self.children))
+    
     def update(self, dt):
         angle = (self.target - Vector(self.width / 2, 0)).angle((0, 100))
         self.angle = angle
-        self.shot.angle = angle
-        self.shot.update(dt)
+        for shot in self.shots:
+            shot.update(dt)
         
     def on_touch_down(self, touch):
-        velocity = self.shot.velocity
-        if (velocity.length2() == 0):
-            self.target = Vector(touch.x, touch.y)
-            v = self.target - Vector(self.width / 2, 0)
-            if (v.length2() > 0):
-                self.shot.velocity = v.normalize() * 10
+        for shot in self.shots:
+            velocity = shot.velocity
+            if (velocity.length2() == 0):
+                self.target = Vector(touch.x, touch.y)
+                v = self.target - Vector(self.width / 2, 0)
+                if (v.length2() > 0):
+                    direction = v.normalize()
+                    shot.shoot(direction)
+                break
 
 class SpawningApp(App):
     def build(self):
         spawning = SpawningScreen()
+        Clock.schedule_once(spawning.start, 0)
```

Comment:
As noted in the README, we need to call start to set up the additional shots _after_ the `kv` file has loaded. This calls the function on the next frame.